logging/dedicated-efk-logging.adoc

```diff
@@ -0,0 +1,65 @@
+:context: dedicated-efk-logging
+[id="dedicated-efk-logging"]
+= About cluster logging and {product-title}
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+As an {product-title} cluster administrator, you can deploy cluster logging
+to aggregate logs for a range of services.
+
+{product-title} clusters come configured with logging via `ElasticSearch`
+operators installed and `Curator` pre-installed to have a two-day retention of
+logs. To utilize these features, create a `ClusterLogging` Custom Resource in
+the `openshift-logging` namespace with the following `.yaml` information:
+
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+metadata:
+  name: "instance"
+  namespace: "openshift-logging"
+spec:
+  managementState: "Managed"
+  logStore:
+    type: "elasticsearch"
+    elasticsearch:
+      nodeCount: 3
+      storage: {}
+      redundancyPolicy: "SingleRedundancy"
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+      resources:
+        request:
+          memory: 8G
+  visualization:
+    type: "kibana"
+    kibana:
+      replicas: 1
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+  curation:
+    type: "curator"
+    curator:
+      schedule: "30 3 * * *"
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+  collection:
+    logs:
+      type: "fluentd"
+      fluentd: {}
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+----
+
+[NOTE]
+====
+To opt-out of logging after it has been installed, delete the `ClusterLogging`
+Custom Resource that was created.
+====
+
+include::modules/efk-logging-about.adoc[leveloffset=+1]
+
+include::modules/efk-logging-about-elasticsearch.adoc[leveloffset=+2]
+
+include::modules/efk-logging-about-curator.adoc[leveloffset=+2]
```

Comment:
Do you want to include the topics about Kibana and Fluentd? They will be installed via the CR (assuming it works the same in Dedicated as it does in Enterprise.

Reply:
I'm not currently aware of Kibana or Fluentd in Dedicated, so I left them out at this time. Thanks!

Reply:
Actually, yes, they are specifically mentioned in the YAML. Good catch! Adding them.

Reply:
Yeah, I assume that the cluster logging will install the Kibana and Fluentd pods as it does in Enterprise.